Assets/Code/Scripts/NoclipRealityManagement/NoclipMovement.cs

```diff
@@ -49,22 +53,23 @@ private void Awake()
     {
         _transform = GetComponent<Transform>();
         _noclipManager = FindObjectOfType<NoclipManager>();
+        _noclipCamera = _transform.GetChild(0).GetComponent<Transform>();
```

Comment:
I am concerned that this could error prone, I think it is difficult to remember that the order in the allPlayer prefab must be fixed. I propose, as an alternative, to give a tag to the noclipCamera and get the child with the noclipCamera tag. [Here there is a code to get the child with a given tag](https://forum.unity.com/threads/how-to-find-child-objects-with-a-specific-tag.1030315/)

This is subjective, also my proposal can be "broken" if someone removes the tag...

Reply:
I put this because the noclip player has only one child and I don't think that other children are needed